readthedocs/projects/migrations/0088_timestamp_migration.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.2.11 on 2022-02-14 16:38
+
+from django.db import migrations
+from django.db.models import Q
+from django.utils import timezone
+
+
+def populate_null_fields(apps, schema_editor):
+    now = timezone.now()
+    query = Q(created__isnull=True) | Q(modified__isnull=True)
+    Project = apps.get_model('projects', 'Project')
+    Project.objects.filter(query).update(
+        created=now,
+        modified=now,
+    )
+
+    ImportedFile = apps.get_model('projects', 'ImportedFile')
+    ImportedFile.objects.filter(query).update(
```

Comment:
The field for this model is `ignore` (boolean), isn't a date that we need to migrate here, similar for Project, the attribute to migrate is `external_builds_privacy_level` (charfield)